docs/sections/data/groups.md

```diff
@@ -71,54 +67,44 @@ PK    Type         Created
  380  CalcJobNode  2078D:17h:46m ago
 1273  CalcJobNode  2078D:18h:03m ago
 ...
-
 ```
 
-Conversely, if you want to see all the groups a node belongs to, you can run:
+Conversely, if you want to see all the groups a node belongs to, you can use its PK and run:
 
 ```{code-block} console
-
 $ verdi group list -a -A --node 380
```

Comment:
I'll use this opportunity to ask you what you think on https://github.com/aiidateam/aiida-tutorials/issues/378 😄